server/graphql/requisition/src/mutations/errors.rs

```diff
@@ -7,3 +7,12 @@ impl CannotDeleteRequisitionWithLines {
         "Cannot delete requisitions with existing lines"
     }
 }
+
+pub struct MaxOrdersReachedForPeriod;
+
+#[Object]
+impl MaxOrdersReachedForPeriod {
```

Comment:
Moved to errors file since both being used in customer and supplier program settings